middleware/packet-forward-middleware/testing/simapp/upgrades/upgrades.go

```diff
@@ -0,0 +1,68 @@
+package upgrades
+
+import (
+	"github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v7/packetforward/keeper"
+	packetforwardtypes "github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v7/packetforward/types"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	consensusparamskeeper "github.com/cosmos/cosmos-sdk/x/consensus/keeper"
+	paramskeeper "github.com/cosmos/cosmos-sdk/x/params/keeper"
+	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+)
+
+const (
+	V2 = "v2"
+)
+
+// CreateDefaultUpgradeHandler creates an upgrade handler which can be used for regular upgrade tests
+// that do not require special logic
```

Comment:
nit: godoc comments should use complete sentences with appropriate punctuation.

Reply:
158652b280b25f197d2b580bbb1a4294b3720641